### PR TITLE
chore: remove forms, http and rxjs from UMD bundle

### LINKED ISSUE #548: UMD bundles contain Angular and RxJS
By quickly checking size of our bundles I've realized that we are bundling Angular and RxJS as part of what gets produced in `dist/bundles`. This is wrong - we should have Angular 2 and RxJS as externals.

### PR DESCRIPTION
**Pull request by Foxandxss**
I used the same technique (more or less) than Angular so it should be good.

If we start adding more operators, I will simplify the setup a bit with a `rxjsExternal` or similar.

Fixes #548

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -74,7 +74,20 @@ gulp.task('umd', function(cb) {
         entry: './dist/index.js',
         output: {filename: 'dist/bundles/ng-bootstrap.js', library: 'ngb', libraryTarget: 'umd'},
         devtool: 'source-map',
-        externals: {'@angular/core': ngExternal('core'), '@angular/common': ngExternal('common')}
+        externals: {
+          '@angular/core': ngExternal('core'),
+          '@angular/common': ngExternal('common'),
+          '@angular/forms': ngExternal('forms'),
+          '@angular/http': ngExternal('http'),
```
Comment (pkozlowski-opensource) [inline gulpfile.js]: We are not using `http` in the library itself (only in the demo) so it is not strictly needed. But it doesn't hurt either so I'm fine with keeping it.

**Comment by pkozlowski-opensource** [comment]
Did you check that no other parts of Angular are pulled in? I saw `compiler` showing up, but maybe it was transitive dependency of forms (would be wired, still)

**Comment by pkozlowski-opensource** [comment]
LGTM otherwise.

File: gulpfile.js (hunk 1 above)
Comment (Foxandxss) [inline gulpfile.js]: Oh, thought we did. Let me fix.

**Comment by Foxandxss** [comment]
Left the terminal open with the fix but never pushed it...

**Pull request merged**